### ISSUE TITLE
create test_compatibility_biosimulators for first example in SBML test suite

### ISSUE THREAD

**Issue by stellaprins** [OpenSourceBrain/SBMLShowcase#49]
Start with the first example (in `00001`) from the SBML test suite and create a `test_compatibility_biosimulators` results table.

- [x] create a `test_00001` subfolder in the `test_suite` folder

`test_00001` folder should contain:
- [x] highest level SBML and SED-ML files (00001)
- [x] `test_compatibility_biosimulators` results table

to do: 
- [x] fix broken plot links 
- [x] remove test_00002 and test_00003 folders before merging

**Comment by stellaprins** [comment]
I'm not aware of any easy way to determine whether a given file is an SBML or SED-ML file (see discussion [here](https://github.com/OpenSourceBrain/SBMLShowcase/issues/31#issuecomment-2192270693)). Previously we decided to use the file extensions to determine the file format (i.e. .sedml and .sbml for SED-ML and SBML files). However, all files in the test suite have .xml extensions. Instead of catogarizing these as pass or FAIL I've decided to categorize compatility (Compat column) as 'unsure' for files with .xml extensions.
